stable/restore/files/nuorestore

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+#
+# simple script to perform a NuoDB restore.
+# Sets a flag to tell newly-starting SM(s) to perform the restore; then optionally causes the SM(s) to restart
+
+# args are:
+#   --type      - retore type: ( database | archive )
+#   --db-name   - database name to perform the restore operation on
+#   --source    - source to restore from - can be a backupset or a URL
+#   --group     - arbitrary group to select the SMs to backup. Often the datacentre or cloud name
```

Comment:
not part of the args while loop

Reply:
Not sure what you mean.
type, db-name, and source are; group is not.